src/components/headerVideo.tsx

```diff
@@ -0,0 +1,46 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css } from '@emotion/core'
+import { remSpace, neutral } from '@guardian/src-foundations';
+
+import { darkModeCss } from 'styles';
+import { Option } from 'types/option';
+import { Video } from 'video';
+
+// ----- Component ----- //
+
+const styles = css`
+    margin: 0 0 ${remSpace[2]} 0;
+    position: relative;
+    display: block;
+    width: 100%;
+    padding-bottom: 56.25%;
+    background: ${neutral[97]};
+    ${darkModeCss`
+        background: ${neutral[20]};
+    `}
+`;
+
+interface Props {
+    video: Option<Video>;
+}
+
+const HeaderVideo: FC<Props> = ({ video }): JSX.Element =>
+    video.fmap(({
+        posterUrl,
+        videoId,
+        duration
+    }) =>
+        <div
+            css={styles}
+            data-posterUrl={posterUrl}
+            data-videoId={videoId}
+            data-duration={duration}
+        >
+        </div>
+    ).withDefault(<></>)
```

Comment:
what does this do? `.withDefault(<></>)`

Reply:
It's almost the same as `.withDefault(null)` but it's to comply with the `JSX.Element` type 

https://reactjs.org/docs/fragments.html

Reply:
An alternative would be to make the return type `ReactElement` and use `null`.